pkg/config/config.go

```diff
@@ -414,6 +414,10 @@ type GitHubMonitor struct {
 	StripSuffix string `json:"strip-suffix,omitempty" yaml:"strip-suffix,omitempty"`
 	// Filter to apply when searching tags on a GitHub repository
 	TagFilter string `json:"tag-filter,omitempty" yaml:"tag-filter,omitempty"`
+	// Filter to apply when searching tags on a GitHub repository
+	TagPrefix string `json:"tag-prefix,omitempty" yaml:"tag-prefix,omitempty"`
+	// Filter to search tags on the Github tag results
+	TagContains string `json:"tag-contains,omitempty" yaml:"tag-contains,omitempty"`
```

Comment:
How would you feel about having `TagFilterPrefix` and `TagFilterContains` be more explicit on what it does?

@jonjohnsonjr - you're good with naming! Thoughts?